CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Snyk Changelog
 
+## [2.8.0]
```

Comment:
This is gonna be fun to merge. One of these days, I'll steal your versioning/releasing solution of code-client and add it here, too ;)